### DISCUSSION TITLE
How to store emoji in database?

### DISCUSSION THREAD

**Discussion by yuvrajtimalsina** [filamentphp/filament#8744] [General]
**I have a scenario where I have to save emoji in database but it doesn't seem to work for some reason.**

```
   Forms\Components\Select::make('birth_country')
                    ->label('Birth Country')
                    ->required()
                    ->native(false)
                    ->placeholder('Select Birth Country')
                    ->default(Countries::np->value)
                    ->options(Countries::class)
                    ->selectablePlaceholder(false),
```
**I have Enum class as**

```
  public function getLabel(): ?string
    {
        return match ($this) {
            self::np => '🇳🇵 Nepal',
            self::us => '🇺🇸 United States',
        };
    }
```

![Screenshot from 2023-09-27 12-09-32](https://github.com/filamentphp/filament/assets/107990171/b7818c3f-5a58-464e-8b72-8e322c7eefd3)

**Comment by VincvanAs** [comment]
Maybe ->allowHtml() helps?

**Reply by yuvrajtimalsina** [reply]
Thanks, but it didn't help. Maybe the enum was not the right choice. I just proceed with helper containing an array and saved it successfully.
